frontend/src/components/requests/filter.jsx

```diff
@@ -17,14 +24,14 @@ export default class RequestsFilter extends React.Component {
         this.submit = this.submit.bind(this);
 
         this.state = {
-            showAdditional: false,
```

Comment:
I switched from always using the state to deriving the values when I needed them. Otherwise it's hard to get the options in the dropdowns to update once we lazy load in the times of day